src/lucky/logger_helpers.cr

```diff
@@ -1,15 +1,16 @@
 module Lucky::LoggerHelpers
-  def self.colored_status_code(status_code : Int32) : String
+  def self.colored_http_status(status_code : Int32) : String
+    status_name = Wordsmith::Inflector.humanize(HTTP::Status.from_value?(status_code) || "")
+    message = "#{status_code} #{status_name}".colorize.bold
+
     case status_code
-    when 200..399
```

Comment:
Most things *should* be green. So rather than distracting with a bold green status, just make it white. That way when a real errors pops up it stands out.

Note: We log halted pipes in yellow so redirect due to incorrect permissions still stands out